src/platform/K32W/KeyValueStoreManagerImpl.cpp

```diff
@@ -24,76 +24,189 @@
 #include <platform/internal/CHIPDeviceLayerInternal.h>
 
 #include <lib/support/CHIPMem.h>
+#include <list>
 #include <platform/K32W/K32WConfig.h>
 #include <platform/KeyValueStoreManager.h>
 #include <string>
 
 #include "PDM.h"
 
+#include <unordered_map>
+
 namespace chip {
 namespace DeviceLayer {
 namespace PersistedStorage {
 
 /* TODO: adjust this value */
-#define MAX_NO_OF_KEYS 255
+#define MAX_NO_OF_KEYS 20
```

Comment:
this is C++, use  `constexpr size_t kMaxNumberOfKeys = 20;`

Reply:
Done, used constexpr, thanks.